README.md

```diff
@@ -8,12 +8,12 @@ This repo is designed to be used as a template for developing new data sets for
 It is currently set up for using `obs_test`.
```

Comment:
Please add a link to and a super brief description of what the `obs_test` package is. Does this mean the dataset template is not intended for use with other obs packages (yet)? Sorry if this is a bit out of scope, this is my first look at this lovely new package 👍

Reply:
This package predates `ap_verify_hits2015`, I'm surprised you didn't know about it. :confused: 

I think the meaning is that the template is configured to use `obs_test` instead of a "real" package like `obs_decam`. Is there a concise way to say that?

Reply:
My mistake, I probably knew about it and forgot... how about "this template dataset is configured for use with `obs_test` (include link), which is itself a framework(template?) for real obs_ packages in the LSST Stack." Perhaps not as concise as you'd like.